init.go

```diff
@@ -6,6 +6,6 @@
 package nottinygc
 
 /*
-#cgo LDFLAGS: -Lwasm -lgc -lmimalloc -lclang_rt.builtins-wasm32 --export=malloc --export=free
+#cgo LDFLAGS: -Lwasm -lgc -lmimalloc -lclang_rt.builtins-wasm32
```

Comment:
Please leave this out of this PR, I am planning on adding a build tag for the behavior since it is specific to proxy-wasm, not TinyGo

Reply:
done